### PR TITLE
Updated speaker link and about.json

### PR DESCRIPTION
**Pull request by Warlord77**

### CODE REVIEW

File: api/about.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
    {
     "logo": "http://bit.ly/2niBg5Y",
     "website":"https://pydelhi.org/",
-    "details":"PyDelhi is a volunteer organization, group of technically inclined students, professionals, teachers and entrepreneurs who want to learn something new having a common primary interest in Python programming language." 
+    "about":"PyDelhi Conf is the annual conference hosted by PyDelhi community.It will be bringing together developers, experts, researchers, scientists, enthusiasts and companies dealing in python.It is a completely volunteer based event, managed by PyDelhi(https://pydelhi.org/) python users group Delhi.On its maiden year PyDelhi Conf attracted a total attendance of 300 people."
```
Comment (CuriousLearner) [inline api/about.json]: Space after `full-stop` when each line ends.

Also, is the key change and text change necessary?

Comment (CuriousLearner) [reply api/about.json]: `Volunteer driven event`, not `volunteer based`.

`Python Delhi Users Group`, not `python users group Delhi`

Comment (shivan1b) [reply api/about.json]: @CuriousLearner Yes. "About" section is supposed to show about the conference not the community. There is a separate section for community.

Comment (CuriousLearner) [reply api/about.json]: @shivan1b Alright, sounds good. Please ensure that they are properly updated on website as well as app. 

Tagging @realslimshanky to ensure that this goes good on website, if needed.

File: api/about.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
  "0.0.1": [
    {
-    "logo": "http://bit.ly/2niBg5Y",
-    "website":"https://pydelhi.org/",
-    "details":"PyDelhi is a volunteer organization, group of technically inclined students, professionals, teachers and entrepreneurs who want to learn something new having a common primary interest in Python programming language." 
+    "logo": "https://paste.opensuse.org/89323686",
```
Comment (shivan1b) [inline api/about.json]: Thanks! All looks good except the logo link. It does not point to an image, please fix that for both the files.

**Review by shivan1b — CHANGES_REQUESTED** [review]

File: api/about.json (hunk 2 above)
Comment (CuriousLearner) [reply api/about.json]: @Warlord77 It should point to raw file like: https://paste.opensuse.org/view/raw/89323686

**Comment by CuriousLearner** [comment]
Looks good now, though quality for Brij's image is less. 

Waiting for @shivan1b 's review on this.

**Review by shivan1b — APPROVED** [review]

**Comment by shivan1b** [comment]
@Warlord77 Thanks!

**Pull request merged**